### PR TITLE
Support for private aliases.

### PR DESCRIPTION
**Pull request by rejinka**
Addresses #21.

### CODE REVIEW

**Comment by rejinka** [comment]
Ah, forgot to add some docs to the readme...

File: src/DefinitionHelper/AliasDefinitionHelper.php [hunk 1]
```diff
@@ -3,6 +3,7 @@
 
 namespace Fluent\DefinitionHelper;
 
+use Symfony\Component\DependencyInjection\Alias;
```
Comment (mnapoli) [inline src/DefinitionHelper/AliasDefinitionHelper.php:6]: Oh I didn't know about that class, nice!

**Comment by mnapoli** [comment]
:+1: I'll wait for the docs but then it's good to merge

**Comment by rejinka** [comment]
@mnapoli done.

**Comment by mnapoli** [comment]
Thanks, sorry for the delay I was focusing my attention on PSR-11 these last days :)

**Pull request merged**